components/demo/demo-snippet.js

```diff
@@ -68,6 +74,23 @@ class DemoSnippet extends LitElement {
 				position: sticky;
 				top: 0;
 			}
+			d2l-dropdown.settings-dropdown {
+				background-color: white;
+				border-radius: 6px;
+				box-shadow: 0 0 0 1px var(--d2l-color-celestine-minus-1);
+				position: fixed;
+				right: 1rem;
+				top: -0.25rem;
+				transition: translate 0.15s, box-shadow 0.15s;
```

Comment:
This is just the demo-snippet so it's not a huge deal, but normally we would layer the transition/animation on with:
```css
@media (prefers-reduced-motion: no-preference) {
	d2l-dropdown.settings-dropdown {
		transition: translate 0.15s, box-shadow 0.15s;
	}
}
```